Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -94,47 +95,56 @@ bool AVIDump::Start(int w, int h)
 
 bool AVIDump::CreateVideoFile()
 {
-  AVCodec* codec = nullptr;
-
-  s_format_context = avformat_alloc_context();
-  std::stringstream s_file_index_str;
-  s_file_index_str << s_file_index;
-  snprintf(s_format_context->filename, sizeof(s_format_context->filename), "%s",
-           (File::GetUserPath(D_DUMPFRAMES_IDX) + "framedump" + s_file_index_str.str() + ".avi")
-               .c_str());
-  File::CreateFullPath(s_format_context->filename);
-
-  // Ask to delete file
-  if (File::Exists(s_format_context->filename))
+  const char* s_format = g_Config.sDumpFormat.c_str();
```

Comment:
This can just be a `const std::string&` which has `.c_str()` called on it where necessary.